extensions/amp-story/1.0/pagination-buttons.css

```diff
@@ -117,6 +113,10 @@
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="48px" height="48px" viewBox="0 0 48 48" fill="#FFFFFF"><path d="M11.76 8.24L27.51 24 11.76 39.76 16 44l20-20L16 4z"/><path fill="none" d="M0 0h48v48H0z"/></svg>') !important;
 }
 
+.i-amphtml-story-fwd-replay {
+  pointer-events: none !important;
+}
+
 .i-amphtml-story-fwd-replay > .i-amphtml-story-button-move {
   background-image: url('data:image/svg+xml;charset=utf-8,<svg xmlns="http://www.w3.org/2000/svg" width="32px" height="32px" viewBox="0 0 24 24"><path d="M17.65 6.35C16.2 4.9 14.21 4 12 4c-4.42 0-7.99 3.58-7.99 8s3.57 8 7.99 8c3.73 0 6.84-2.55 7.73-6h-2.08c-.82 2.33-3.04 4-5.65 4-3.31 0-6-2.69-6-6s2.69-6 6-6c1.66 0 3.14.69 4.22 1.78L13 11h7V4l-2.35 2.35z"/><path d="M0 0h24v24H0z" fill="none"/></svg>') !important;
```

Comment:
Can't comment on the exact line but, given that these buttons will be present in mobile / full-bleed now. I think we should hide the background image when in mobile, right?

https://github.com/ampproject/amphtml/blob/cf751b27a6334791e55220e4629199f61ad64abe/extensions/amp-story/1.0/pagination-buttons.css#L93-L113

Reply:
Definately.
`.i-amphtml-story-button-move` is `display: none` by default. The button is only visible in desktop modes.
https://github.com/ampproject/amphtml/blob/fa8a10868002ed721c4f70c84b7e23e949a69ed8/extensions/amp-story/1.0/pagination-buttons.css#L53-L56

Reply:
Got it, thx